admin-dev/themes/new-theme/js/types/vue/shim.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module '*.vue' {
```

Comment:
Not sure what is the usage of this, but maybe it's worth adding a comment for future developers to understand it